### DISCUSSION TITLE
无法登录服务器（验证选项内清除了其它选项）

### DISCUSSION THREAD

**Discussion by guguailf** [kingToolbox/WindTerm#1145] [General]
其它ssh工具能够登录，但是windterm无法登录，已经将该会话的验证选项内的其它选项去掉也不行，请教各位谁遇到过这样的情况？
![图片](https://user-images.githubusercontent.com/56786055/187188315-75a52cca-c162-4e22-b166-0beb50b88cf5.png)

**Comment by guguailf** [comment]
已经处理完毕
本问题无需回答，谢谢

**Reply by John17Wang** [reply]
请问怎么解决的呢？我也有这个问题
